### PR TITLE
changed to api.core-os.net and added certs

### PR DESCRIPTION
**Pull request by doodles526**
tests confirm working functionality with ./core-admin ping

### CODE REVIEW

File: new-version.go [hunk 1]
```diff
@@ -143,7 +143,10 @@ func runNewVersion(cmd *Command, args []string) {
 		return
 	}
 
-	client := http.Client{}
+	client := &http.Client{
+		Transport: tlsTransport,
```
Comment (doodles526) [inline new-version.go:147]: Do we want to do it this way, or do we just wanted a certed client to be declared in const?

Comment (philips) [inline new-version.go:147]: please declare the certs a const. you can build the constants file like we do in etcd: https://github.com/coreos/etcd/blob/master/web/file2gostring.sh

File: const.go [hunk 2]
```diff
@@ -21,4 +29,31 @@ func init() {
 		}
 		updateURL = *envUrl
 	}
+	pool := x509.NewCertPool()
+	
+	coreosInetPemBlock, _ := pem.Decode([]byte(certs.CoreOS_Internet_Authority_pem))
+	if coreosInetPemBlock == nil {
+		fmt.Printf("Error: No PEM data found in CoreOS_Internet_Auth")
+		os.Exit(-1)
```
Comment (philips) [inline const.go]: since we are decoding from the binary now it might make more sense to just panic()

**Comment by philips** [comment]
Looks great, just that one nit about using panic() instead of printf and os.exit. Also, just as a side note usually os.Exit return codes are uint8 so they should be a non-zero positive number on error.

**Pull request merged**